packages/web/src/assets/book-terms/README.md

```diff
@@ -0,0 +1,21 @@
+# Overview
```

Comment:
All of these notes, we will want to move to the localization docs

Reply:
Note for self: still needs done

Reply:
Moved